ui/src/features/Editor/components/OverlayUI/components/NodePickerDialog/index.tsx

```
@@ -113,11 +152,12 @@ const NodePickerDialog: React.FC<Props> = ({
           autoFocus
           onChange={(e) => handleSearch(e.target.value)}
         />
-        <div className="max-h-[50vh] overflow-scroll">
+        <div ref={containerRef} className="max-h-[50vh] overflow-scroll">
           {actions?.map((action, idx) => (
             <Fragment key={action.name}>
               <ActionItem
-                className="m-1"
+                ref={(el) => (itemRefs.current[idx] = el)}
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)